pkgs/os-specific/linux/nfs-utils/default.nix

```diff
@@ -56,6 +56,18 @@ stdenv.mkDerivation rec {
       url = "https://raw.githubusercontent.com/alpinelinux/aports/cb880042d48d77af412d4688f24b8310ae44f55f/main/nfs-utils/musl-getservbyport.patch";
       sha256 = "1fqws9dz8n1d9a418c54r11y3w330qgy2652dpwcy96cm44sqyhf";
     })
+    (fetchpatch {
```

Comment:
```suggestion
    # fixes for musl
    (fetchpatch {
```

Reply:
I would've thought line 53:
`patches = lib.optionals stdenv.hostPlatform.isMusl`
made the purpose of the changes clear? This is just additional patches for the same purpose as the original.

Reply:
Oh, yes that also works.